static/js/components/tiles/ranking_tile.tsx

```diff
@@ -71,6 +77,8 @@ export function RankingTile(props: RankingTilePropType): JSX.Element {
   const [rankingData, setRankingData] = useState<RankingData | undefined>(null);
   const embedModalElement = useRef<ChartEmbed>(null);
   const chartContainer = useRef(null);
+  const rankingUnitHighestContainer = useRef<HTMLDivElement>(null);
```

Comment:
a ranking tile can have mulitple ranking units for highest & multiple ranking units for lowest (e.g., https://screenshot.googleplex.com/BbnFKQFz6LYN3FZ) .. we will probably need a ref for each ranking unit?